### PR TITLE
Travis refactorings #37210

### PR DESCRIPTION
**Pull request by thde**
The build currently fails because some tests fail apparently.

### CODE REVIEW

**Comment by cimnine** [comment]
I believe the tests have been fixed in https://github.com/ninech/django-netbox-graphql/pull/7/commits/c64e3a358f1605d0cbdb40d995ace78f870f05ed.

**Comment by thde** [comment]
@cimnine, I tried it and got the following results:

* `dc exec netbox ./manage.py test netbox-graphql/` OK
* `dc exec netbox ./manage.py test netbox-graphql/ -r` (reversed order) 22 failing
* `dc exec netbox ./manage.py test` 19 failing
* `dc exec netbox ./manage.py test -r` 32 failing 

They seem not independent to me. But I'll try to fix them if I can manage to understand what's going on 😄

**Comment by thde** [comment]
The tests are now redone and work together with the Netbox tests. I see still some room for improvement though (but maybe in another PR):
* family and status request int as input but return a string (4 -> 'A_4')
* mutations could be named diffenently: http://graphql.org/learn/queries/#mutations

File: README.rst [hunk 1]
```diff
@@ -152,7 +152,7 @@ Tests
 -----
 Run unit tests::    
     docker-compose run netbox ./manage.py test # runs all tests
-    docker-compose run netbox ./manage.py test netbox-graphql/ # runs only netbox-graphql module tests
+    docker-compose run netbox ./manage.py test netbox_graphql/ # runs only netbox_graphql module tests
```
Comment (cimnine) [inline README.rst]: Not your fault, but the formatting is weird here: https://github.com/ninech/django-netbox-graphql/blob/a12d515a8fca2fbc9eaf5ee70814d80cbe3fd394/README.rst#tests

**Comment by cimnine** [comment]
Also, please create issues on the internal issue tracker (and maybe on the Github issue tracker as well) for your suggested improvements:

> * family and status request int as input but return a string (4 -> 'A_4')
> * mutations could be named diffenently: http://graphql.org/learn/queries/#mutations

File: README.rst (hunk 1 above)
Comment (thde) [reply README.rst]: 👍 done

**Pull request merged**